catalog/core/catalog-core-versioning/versioning-common/src/main/java/ddf/catalog/core/versioning/MetacardVersion.java

```diff
@@ -186,7 +213,17 @@ public static Action ofMetacard(Metacard metacard) {
      * @throws IllegalArgumentException
      */
     public MetacardVersion(Metacard sourceMetacard, Action action, Subject subject) {
-        super(sourceMetacard, metacardVersion);
+        this(sourceMetacard, action, subject, Collections.singletonList(BasicTypes.BASIC_METACARD));
+    }
+
+    public MetacardVersion(Metacard sourceMetacard, Action action, Subject subject,
+            List<MetacardType> types) {
+        super(sourceMetacard,
+                new MetacardTypeImpl(VERSION_PREFIX,
+                        getMetacardVersionType(),
```

Comment:
I think it might be clearer in this instance to just directly use the `metacardVersion` static instead of invoking this method.